stages/test/test_kickstart.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/python3
+
+import os.path
+import tempfile
+from dataclasses import dataclass
+
+import pytest
+
+from osbuild.testutil.imports import import_module_from_path
+
+
+@dataclass
+class KsTestCase:
```

Comment:
Not sure if this is overdoing it TBH but the alternative would be just using a dict

Reply:
Due to our `>= 3.6` constraint for Python we can't use dataclasses :(